lib/school_house_web/controllers/post_controller.ex

```diff
@@ -23,6 +23,12 @@ defmodule SchoolHouseWeb.PostController do
     end
   end
 
+  def filter_by_tag(conn, %{"tag" => tag}) do
+    with {:ok, posts_by_tag} <- Posts.get_by_tag(tag) do
+      render(conn, "tag.html", page_title: @page_title, posts: posts_by_tag, pages: Posts.pages(), tag: tag)
+    end
+  end
```

Comment:
Rather than creating a new template for this page (`tag.html.eex`), how about using the `index.html` template and passing through another property on `assigns` for the page name. You'd just need to update the `templates/post/index.html.eex`.